feature-configs/deploy/gatekeeper/is_ready.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+oc -n gatekeeper-system wait deployment/gatekeeper-controller-manager --for condition=available --timeout 5s
```

Comment:
please wait for only 1s here the deploy loop will retry until it's ready so no need to wait 5s here